charts/opentelemetry-kube-stack/examples/cloud-demo/values.yaml

```diff
@@ -0,0 +1,42 @@
+clusterName: demo
+opentelemetry-operator:
+  enabled: true
+defaultCRConfig:
+  presets:
+    otlpExporter:
+      enabled: true
+      endpoint: ingest.example.com:443
+      headers:
+        "access-token": "${ACCESS_TOKEN}"
+collectors:
+  daemon:
+    env:
+      - name: ACCESS_TOKEN
+        valueFrom:
+          secretKeyRef:
+            key: access_token
+            name: otel-collector-secret
+  cluster:
+    env:
+      - name: ACCESS_TOKEN
+        valueFrom:
+          secretKeyRef:
+            key: access_token
+            name: otel-collector-secret
+
+instrumentation:
+  enabled: true
+  env:
+    - name: OTEL_K8S_NODE_NAME
```

Comment:
I'm not 100% sure doing this works for the SDKs... I haven't tested this but i think it would work!